### PR TITLE
OSDOCS-2626 - Adding HTPasswd to the IdP docs for OSD

### PR DESCRIPTION
**Pull request by pneedle-rh**
This applies to `dedicated-4` only.

This relates to https://issues.redhat.com/browse/OSDOCS-2626. The PR adds HTPasswd to the list of supported identity providers in the OSD docs. A section on "Configuring an HTPasswd identity provider" is also added.

The preview is [here](https://deploy-preview-36491--osdocs.netlify.app/openshift-dedicated/latest/identity_providers/config-identity-providers.html).

### CODE REVIEW

**Comment by pneedle-rh** [comment]
@vkareh can you please review this PR from an SME perspective? Thanks!

**Comment by pneedle-rh** [comment]
@xueli181114 can you please review this PR from a QE perspective? Thanks!

**Comment by xueli181114** [comment]
@yuwang-RH  Would you please review the doc change?

**Comment by vkareh** [comment]
LGTM

File: modules/config-htpasswd-idp.adoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+// * assemblies/config-identity-providers.adoc
+
+[id="config-htpasswd-idp_{context}"]
+= Configuring an HTPasswd identity provider
+
+Configure an HTPasswd identity provider to create a single, static user with cluster administration privileges. You can log in to your cluster as the user to troubleshoot issues.
+
+.Procedure
+
+. From the link:https://console.redhat.com/[OCM console], navigate to the *Clusters* page and select the cluster that you need to configure identity providers for.
```
Comment (kalexand-rh) [inline modules/config-htpasswd-idp.adoc:12]: Please double-check this use of "OCM console." I see "{cloud-redhat-com}" and "console.redhat.com" a lot, "{cloud-redhat-com} (OCM)" less frequently, and one "OCM" by itself after the initialism was defined in the module.

File: modules/config-htpasswd-idp.adoc [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+. Select *HTPasswd* from the *Identity Provider* drop-down menu.
+
+. Add a unique name in the *Name* field for the identity provider.
+
+. Use the suggested username and password for the static user, or create your own.
++
+[NOTE]
+====
+The credentials defined in this step are not visible after you select *Confirm* in the following step. If you lose the credentials you must recreate the identity provider and define the credentials again.
```
Comment (kalexand-rh) [inline modules/config-htpasswd-idp.adoc:31]:
```suggestion
The credentials defined in this step are not visible after you select *Confirm* in the following step. If you lose the credentials, you must recreate the identity provider and define the credentials again.
```

**Review by kalexand-rh — COMMENTED** [review]
This is looking really good! It's almost ready to go.

File: modules/config-htpasswd-idp.adoc (hunk 1 above)
Comment (pneedle-rh) [reply modules/config-htpasswd-idp.adoc]: @kalexand-rh thank you for your review and for highlighting this.

I have updated the line in this module and the other similar modules included in this assembly to use the `{cloud-redhat-com}` attribute instead. That attribute links to https://cloud.redhat.com/openshift, which now redirects to https://console.redhat.com/openshift. So, although the attribute directs the reader to the correct page ultimately, I think we need to discuss what to do with it in the broader sense as a team. For instance, should we just update the URL in the attribute, or change the attribute throughout the library to `:console-redhat-com:` also? I will add this to our team meeting agenda.

Comment (kalexand-rh) [reply modules/config-htpasswd-idp.adoc]: Thank you, @pneedle-rh! Someone might need to volunteer to do a larger scrub to standardize our usage, and it'll be good to talk about it on the team meeting.

File: modules/config-htpasswd-idp.adoc (hunk 2 above)
Comment (pneedle-rh) [reply modules/config-htpasswd-idp.adoc]: Thank you! Resolved.

**Pull request merged**